wordpress-mobile/WordPress-iOS
# Media Notices: Different titles for notifications

This PR updates the media notices implementation to match our designs, with different text for Notices vs Notifications.

![notice-notification-text](https://user-images.githubusercontent.com/4780/35526368-17cfd3b4-051f-11e8-901c-cdeaa3099d2c.png)

**To test:**

* Test both successful and failed media uploads, and check the text of notices (displayed in the app) and notifications (displayed outside of the app). Check they match the screenshot above.
